server/handlers/postSurveys.js

```diff
@@ -41,7 +41,7 @@ const postSurveys = async (req, res) => {
   //       .min(5)
   //       .max(140)
   //       .required(),
-  //     status: Joi.string().valid('created', 'published', 'closed'),
+  //     status: Joi.string().valid('created', 'active', 'closed'),
```

Comment:
Lots of uncommented code here?

Reply:
not mine

Reply:
This is a joi schema that would only be valid once the survey editor page is finalised (so we know exactly what we're sending to the backend). I'll uncomment it and fix it up at some point today.